Gemfile.lock

```diff
@@ -190,6 +190,7 @@ GEM
     launchy (2.5.0)
       addressable (~> 2.7)
     libsqreen (1.0.4.0.0)
+    libv8-node (16.10.0.0)
     libv8-node (16.10.0.0-x86_64-linux)
```

Comment:
Maybe it's worth removing all linux-specific dependencies. I'll try to do that now.